### PR TITLE
{compiler}[dummy] GCCcore 7.1.0

### PR DESCRIPTION
**Pull request by boegel**

### CODE REVIEW

File: easybuild/easyconfigs/g/GCCcore/mpfr-3.1.5-allpatches-20161219.patch [hunk 1]
```diff
@@ -0,0 +1,176 @@
+# MPFR v3.1.5 patch dated 2017-12-19
```
Comment (wpoely86) [inline easybuild/easyconfigs/g/GCCcore/mpfr-3.1.5-allpatches-20161219.patch]: 2016

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2441.golett.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/1803a7e15e59caa56d2e89dd85228b96 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
node2123.delcatty.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/882778e68d867ccdd460e812d630a2b4 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2003.delcatty.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/3ec97006f84b87c9cc96537cae0fa812 for a full test report.

**Comment by boegel** [comment]
Thanks for the review @wpoely86!

**Pull request merged**